docs/getting_started/limitations.md

```diff
@@ -16,9 +16,15 @@ The Cloud Foundry command line client has an option to enable you to connect dir
 
 This functionality is currently disabled on the Government PaaS. We intend to make it available in future, following security testing.
 
-## 404s after push or restart
+## 404s after commands that restart the app
 
-After you use ``cf push`` or ``cf restart``, your app may briefly return incorrect 404 errors. Apart from the brief downtime, this may lead to problems if the 404 is cached, or visiting web crawling bots (as used by search engines) receive a 404.
+After you use a command that restarts application instances, such as ``cf push`` or ``cf restart``, your app may briefly return incorrect 404 errors. Apart from the brief downtime, this may lead to problems if the 404 is cached, or visiting web crawling bots (as used by search engines) receive a 404.
```

Comment:
Fair point, however it already said that *before* this pull request and I think it's clear enough to the users what we mean, so I think we should leave it in unless we see that it confuses people.